src/steamship/agents/schema/context.py

```diff
@@ -38,6 +39,14 @@ def id(self) -> str:
     """Called when an agent execution has completed. These provide a way for the AgentService
     to return the result of an agent execution to the package that requested the agent execution."""
 
+    action_cache: ActionCache
```

Comment:
nit: I think this and llm_cache below technically should be `Optional[]` types since they're None-able